azurelinuxagent/common/protocol/wire.py

```diff
@@ -1304,6 +1320,70 @@ def parse(self, xml_text):
         return self
 
 
+class RemoteAccess(object):
```

Comment:
Do not recall whether we discussed if this feature exists or will exist in Azure Stack?

Reply:
Consider moving these new classes out of wire.py.

Reply:
I can, but this seems to match the existing code style the best.

---
In reply to: [194029368](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194029368) [](ancestors = 194029368)

Reply:
No, we haven't discussed anything about this.

---
In reply to: [194029271](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194029271) [](ancestors = 194029271)

Reply:
This needs to be clarified. What is the expectation for stack? Will there be a JIT implementation?

Reply:
None.  JIT is not part of Azure Stack today as we're internal only.  Long-long term it is an objective, but nothing remotely near.

---
In reply to: [194177895](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194177895) [](ancestors = 194177895)

Reply:
Ok, thanks for clarifying